app/src/main/java/org/fossasia/openevent/app/contract/view/LoginView.java

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.app.contract.view;
+
+public interface LoginView {
+
+    void showProgressBar(boolean show);
```

Comment:
I am saying progressbar **can be called by the view itself** when it will call presenter's login method and can be dismissed in the onSuccess or onError functions. Hence **will not require to be called by presenter**.
Hence need not to be **implemented** in the view.